src/cpp/ext/otel/key_value_iterable.h

```diff
@@ -132,8 +151,10 @@ class OpenTelemetryPlugin::KeyValueIterable
       additional_labels_;
   const OpenTelemetryPlugin::ActivePluginOptionsView*
       active_plugin_options_view_;
-  absl::Span<const std::shared_ptr<std::map<std::string, std::string>>>
+  std::map<grpc_core::ClientCallTracer::CallAttemptTracer::OptionalLabelKey,
```

Comment:
We definitely do not want to make a copy of this map -- that's needless allocation.  I think this should go back to using `absl::Span<>`.

Reply:
uh yeah, definitely don't want to make a copy